botocore/docs/bcdoc/restdoc.py

```diff
@@ -16,7 +16,23 @@
 from botocore.docs.bcdoc.docstringparser import DocStringParser
 from botocore.docs.bcdoc.style import ReSTStyle
 
+LARGE_SECTION_MESSAGE = (
+    '    ::\n\n        # This section is too large to render.'
+    '\n        # Please see the AWS API Documentation linked below.\n\n    '
```

Comment:
nit; if this still renders correctly, it may be nicer formatting in the code. Also do we need the trailing spaces for something?

```suggestion
    '    ::\n\n'
    '        # This section is too large to render.\n'
    '        # Please see the AWS API Documentation linked below.\n\n'
```

Reply:
Thanks! I'll use this to have nicer formatting. The trailing spaces are needed so the link thats added later is indented the same amount as this code block.

Reply:
Personal preference but I would recommend using triple quotes here so you can write the string literally without the new line characters.  You'll just probably have to use `textwrap.dedent` to remove any unwanted indentations.